css/styles.css

```diff
@@ -382,3 +382,161 @@ section.padded {
   margin-bottom: 8px;
   color: #555;
 }
+
+.categories {
+  display: block;
+  position: relative;
+  text-align: center;
+  font-weight: bolder;
+  font-size: 14px;
+}
+
+.tab-1 {
+  margin-right: 30px;
+}
+
+.tab-2 {
+  margin-left: 30px;
+}
+
+.categories a:hover {
+  color: #4c884c;
+}
+
+.categories a:active {
+  color: #4c884c;
+}
+
+.tab-container1 {
+  display: block;
+  padding-left: 16px;
+  padding-right: 16px;
+  max-width: 1200px;
+  margin: 0 auto;
+  padding-top: 32px;
+  padding-bottom: 32px;
+}
+
+.bestsellers {
+  display: flex;
+  justify-content: space-between;
+  flex-flow: row wrap;
+}
+.bestsellers::after {
```

Comment:
Why is it required?

Reply:
This is for second row elements spacing. Without this the last two items are not spaced properly.